### PR TITLE
Pipeline patch

### PR DESCRIPTION
**Pull request by halles**

### CODE REVIEW

**Comment by halles** [comment]
@phptek this fixes the CI for master by explicitly requesting mysql as a service and changes a test that would fail if database is not empty (ie. development environments). This will enable the other PRs to be merged

File: .travis.yml [hunk 1]
```diff
@@ -12,22 +12,22 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DB=MYSQL
-    - php: 7.0
-      env:
```
Comment (phptek) [inline .travis.yml]: Happy to lose-out on 5.x, and I appreciate 7.0 is EOL, but SilverStripe itself still works on 7.0 and there's nothing particular about the module that precludes it from running on 7.0 - can it be made to work? (I know how much of a PIA it is to work with Travis btw! :-)

Comment (phptek) [reply .travis.yml]: FWIW< the README currently stipulates <7.2, which should prob be updated to reflect whatever we decide to do here. Thanks!

Comment (halles) [reply .travis.yml]: @phptek True. I have no opinion on which minimum version tbh. We don't and won't develop or run on anything lower than 7.2 from now on as 7.1 hits EOL in a few months.

The composer file should reflect this as well (and is currently at =<7.2).

Do you want me set tests/composer/readme from 5.6 til 7.3? 7.0 to 7.3? Or any other combination?

Comment (phptek) [reply .travis.yml]: 7.0+ please. I can update the docs  myself if you like?

Comment (halles) [reply .travis.yml]: @phptek already changed it. Had not read your message

**Comment by halles** [comment]
@phptek let me know if I can squash and push after these edits :)

**Comment by phptek** [comment]
Looks great! Squash and push away!

**Comment by halles** [comment]
Thanks for the supports @phptek. With these we'll be able to deploy and also have a production project where to test the alpha branch so it can progress too :)

**Pull request merged**